javascript/javascript1/week3/homework/script.js

```diff
@@ -0,0 +1,28 @@
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  const borrar = names.indexOf(nameToRemove);
+
+  
+  // Write some code here
+  
+  if (names.includes(nameToRemove)) {
+
+     names.splice(borrar, borrar);
+
+    }
+
+
```

Comment:
How ever if you want it to be fancy and return the same result, instead of `if statement`:

```suggestion
function nameToRemove(name) {
  names.splice(names.indexOf(name), 1);
}
```

And you can also try this
it might be fun ;)
```suggestion
function nameToRemove(name) {
  names.splice(names.indexOf(name), 1, "Simon");
}
```